app/assets/javascripts/services/post_service.js

```diff
@@ -0,0 +1,63 @@
+ManageIQ.angular.app.service('postService', ["miqService", "$timeout", "$window", function(miqService, $timeout, $window) {
+
+  this.saveRecord = function(apiURL, redirectURL, updateObject, successMsg) {
+    miqService.sparkleOn();
+    return API.post(apiURL,
+      angular.toJson({
+        action: "edit",
+        resource: updateObject
+      })).then(handleSuccess, handleFailure);
+
+    function handleSuccess(response) {
+      $timeout(function () {
+        $window.location.href = redirectURL + '&flash_msg=' + successMsg;
+        miqService.sparkleOff();
+        miqService.miqFlash("success", successMsg);
```

Comment:
Aah, these 2 last lines lead to that state where it *looks* like it's done, shows a flash message and then reloads the page with the flash message shown again .. maybe just keep the `href` line, so that we sparkle until the new page is loaded..? 

I'd say same for failure, but I'm not sure we should redirect somewhere else on failure... Shouldn't we just stay on the same screen and remove the `href` line in `handleFailure`?